Source/AnimatableImageView.swift

```diff
@@ -1,15 +1,19 @@
-import ImageIO
 import Runes
 import UIKit
 
 /// A subclass of `UIImageView` that can be animated using an image name string or raw data.
-public class AnimatableImageView: UIImageView, Animatable {
+public class AnimatableImageView: UIImageView {
   /// An `Animator` instance that holds the frames of a specific image in memory.
-  var animator: Animator?
+  private var animator: Animator?
+  /// A display link that keeps calling the `updateFrame` method on every screen refresh.
+  private lazy var displayLink: CADisplayLink = CADisplayLink(target: self, selector: Selector("updateFrame"))
+
+  /// The size of the frame cache.
+  public var frameCacheSize = 50
```

Comment:
Cache is when you load something and persist it for further reuse. I think we are talking about `preload` here; when you load something ahead of its use. How do you feel abut `framePreloadCount`?